### DISCUSSION TITLE
404 error clicking published wallet

### DISCUSSION THREAD

**Discussion by neri-ripio** [WalletConnect/walletconnect-monorepo#2535] [Developer Support]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/129459236/1424805f-c827-44f7-80f2-335a0ef4c6ee)

https://walletconnect.com/explorer/ripio-portal-mobile-wallet

**Comment by finessevanes** [comment]
I've approved the following project. UUID: `38263eaf-53de-43c9-9722-29abc01a3820`. Let me know if this resolves your error.

**Comment by neri-ripio** [comment]
not yet... still showing 404

**Reply by finessevanes** [reply]
Ok, I will let our team know.

**Reply by finessevanes** [reply]
@neri-ripio LGTM! 
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/44731696/008747d1-07cb-4f15-8fb4-c10bae07c6d4)

If this is good to go, please close this discussion! :)
